backend/api/quivr_api/modules/brain/integrations/Multi_Contract/Brain.py

```diff
@@ -115,7 +116,7 @@ def __init__(
     def get_chain(self):
 
         list_files_array = (
-            self.knowledge_qa.knowledge_service.get_all_knowledge_in_brain(
+            await self.knowledge_qa.knowledge_service.get_all_knowledge_in_brain(
```

Comment:
Function is not async. 

This is catched by linter 100% of the time. Maybe force the linter configuration in IDE